packages/pods-core/src/builtin/NextjsExportPod.ts

```diff
@@ -270,7 +270,21 @@ export class NextjsExportPod extends ExportPod<NextjsExportConfig> {
     );
     const podDstPath = path.join(podDstDir, "notes.json");
     const podConfigDstPath = path.join(podDstDir, "dendron.json");
-    fs.writeJSONSync(podDstPath, payload, { encoding: "utf8", spaces: 2 });
+    fs.writeJSONSync(
+      podDstPath,
+      {
+        ...payload,
+        notes: Object.fromEntries(
+          Object.entries(payload.notes).map(([idx, { body: _body, ...note }]) => [
```

Comment:
This is a little hard to read. Lodash's `omit` function would be much simpler here: https://lodash.com/docs/4.17.15#omit

Reply:
+1 on using `_.omit`

Reply:
`_.omit` can only replace object destructing, you actually need `_.mapValue` and `_.omit`. I agree it doesn't look as clean, however I'm leaning towards refactor it for readability while using native methods as long as it's just one or two lines.